projects/go-compile-instrumentation.md

```diff
@@ -0,0 +1,65 @@
+
+This SIG will focus on:
+
+- Developing compiler plugins or enhancements that inject instrumentation code automatically, ensuring minimal runtime performance overhead and compatibility with existing Go projects.
+- Providing standardized instrumentation patterns aligned with OpenTelemetry and other monitoring frameworks.
+
+We want to ensure:
+
+- Comprehensive Coverage: Automatic instrumentation covers common frameworks and libraries used in Go applications, fully compatible with OpenTelemetry’s APIs and SDKs.
+- Performance Efficiency: Instrumentation introduces minimal runtime overhead, maintaining the high-performance standards of Go applications while utilizing OpenTelemetry’s optimized data collection mechanisms.
+- Ease of Use: Developers can enable instrumentation with simple/zero configuration changes without manual code modifications.
+- Extensibility: The instrumentation framework can be extended to support new libraries and frameworks as they emerge in the Go ecosystem, ensuring ongoing compatibility with the evolving OpenTelemetry landscape.
+- Flexibility: Developers are enabled to control their tracing experience (i.e, specifying custom span tags in certain places; opting certain code paths or instances out of tracing, etc...), which can be delivered via a declarative/programmable scheme.
+
+By closely aligning with OpenTelemetry, the Go Compile Instrumentation SIG ensures that Go applications benefit from standardized, vendor-neutral, high-quality observability solutions that are both robust and easy to implement. Having one single, standard tool removes decision points from prospective developers, which makes the path to observability shorter & easier.
+
+## Deliverables
+
+- A flexible and extensible instrumentation framework for Go at compile time
+- Out-of-box instrumentation for common libraries and frameworks in Go application
+- Comprehensive documentation that covers a wide array of topics, including how to use the framework, configuration options, advanced usage scenarios, and answers to FAQs
+
+## Staffing/Help Wanted
+The following vendors are interested in improving this area:
+
+- Alibaba
+- Datadog
+- QuesmaOrg
+
+We also welcome everyone interested in this area to participate in the discussion.
+
+### Required staffing
+
+- Project Lead: @ralf0131(Alibaba Cloud), @dineshg13(Datadog), @pdelewski(QuesmaOrg)
+- Sponsoring GC Members: @jpkrohling
+
+Additionally, the following people will participate in the SIG and be added as approvers once they are OpenTelemetry Github org members:
+
+- Future Approvers: @yiyang0(Alibaba Cloud), @123liuziming(Alibaba Cloud), @RomainMuller(Datadog)
+
+## Meeting Times
```

Comment:
do you think it's feasible to start already with rotating meeting times like the collector has? this way, most people will be able to attend 2/3 of the sessions

Reply:
I am leaning towards start the first couple round of meeting, and adjust if there is anyone who is interested. Of course, other comments are welcome.